content/en/jobs/index.html

```diff
@@ -258,24 +258,7 @@ <h1>Join the Pocket Team</h1>
               <div class="jobs-listings">
                 <section>
                   <h5>Current Positions</h5>
-                  <ul class="jia2zb6">
-                    <li>
-                      <a
-                        href="https://boards.greenhouse.io/pocketco/jobs/2603449"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                        >Senior Software Engineer, Backend</a
-                      >
-                    </li>
-                    <li>
-                      <a
-                        href="https://boards.greenhouse.io/pocketco/jobs/2436558"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                        >Staff Software Engineer - iOS</a
-                      >
-                    </li>
-                  </ul>
+                  <p><a href="https://careers.mozilla.org/listings/?team=Pocket" target="_blank" rel="noopener noreferrer">Click here</a> to see a list of positions we are hiring for.</p>
```

Comment:
Can we avoid "click here" as link text?
```suggestion
                  <p><a href="https://careers.mozilla.org/listings/?team=Pocket" target="_blank" rel="noopener noreferrer">See our latest job openings</a></p>
```